src/llvm-final-gc-lowering.cpp

```diff
@@ -225,6 +207,23 @@ void FinalLowerGC::lowerGCAllocBytes(CallInst *target, Function &F)
     target->eraseFromParent();
 }
 
+void FinalLowerGC::lowerQueueGCRoot(CallInst *target, Function &F)
+{
+    ++QueueGCRootCount;
+    assert(target->arg_size() == 1);
+    target->setCalledFunction(queueRootFunc);
+}
+
+void FinalLowerGC::lowerSafepoint(CallInst *target, Function &F)
+{
+    ++SafepointCount;
+    assert(target->arg_size() == 1);
+    IRBuilder<> builder(target);
+    Value* signal_page = target->getOperand(0);
+    builder.CreateLoad(T_size, signal_page, true);
+    target->eraseFromParent();
+}
+
```

Comment:
Seems like just a cosmetic change? I suggest dropping this change to simplify an already large PR.

Reply:
It's probably a vestigial change from when I was considering changing a few things in codegen for the interface.

Will remove it.